### PR TITLE
Adding prometheus exporter

### PR DESCRIPTION
**Pull request by BetaFold3**
This should add prometheus exporter for pktvisor

### CODE REVIEW

File: pktvisor_exporter/pktvisor_exporter_test.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2016, 2017 Nicolas Lamirault <nicolas.lamirault@gmail.com>
```
Comment (weyrick) [inline pktvisor_exporter/pktvisor_exporter_test.go]: assuming this is copy paste, please fix

**Review by weyrick — CHANGES_REQUESTED** [review]
Thanks for this! Since there may be other exporters, can you please rename it from `pktvisor_exporter` to `pktvisor_prometheus` or similar? And then please put it under the `reporting/` directory for now, where we can start to organize tools and documentation for central collection and visibility.

File: pktvisor_exporter/pktvisor_exporter_test.go (hunk 1 above)
Comment (tnguyenns1) [reply pktvisor_exporter/pktvisor_exporter_test.go]: @weyrick All fixed.

Comment (BetaFold3) [reply pktvisor_exporter/pktvisor_exporter_test.go]: @weyrick All fixed.

**Review by weyrick — APPROVED** [review]

**Pull request merged**